devops/Makefile

```diff
@@ -34,10 +34,50 @@ define API
 "api-$(APP_NAME)"
 endef
 
-define Web
+define WEB
 "web-$(APP_NAME)"
 endef
 
+define WEB_PORT
+7070
+endef
+
+define API_PORT
+3030
+endef
+
+define DATABASE_NAME_KEY
+"database-name"
+endef
+
+define QA_API
```

Comment:
The declaration of the specific QA variables does not seem to be following the DEV/TEST deployment.
Can we not handle the variables in .env file for local, or git hub actions?